ultra/tests/test_analysis.py

```diff
@@ -54,6 +54,7 @@ def test_interface_analyze(self):
                 stopwatcher_route="south-west",
```

Comment:
Are the maps with the extra road segment necessary for analysis? I forgot to ask in the meeting, but what did Kimia say about this?

It would be ideal if we could perform the analysis on the same maps we are experimenting on... Do you see any way of doing this while still getting rid of the extra road segment on the maps?